contrib/akamai/controlbar/ControlBar.js

```diff
@@ -495,7 +503,7 @@ var ControlBar = function (dashjsMediaPlayer, displayUTCTimeCodes) {
 
                 if (availableTracks.audio.length > 1 || availableTracks.video.length > 1) {
                     contentFunc = function (element) {
-                        return 'Language: ' + element.lang + ' - Role: ' + element.roles[0];
+                        return 'Language: ' + (element.label || element.lang) + ' - Role: ' + element.roles[0];
```

Comment:
Hmmmmm. The label isn't guaranteed to be a human-friendly language -- such as "stereo" or "5.1 surround" -- so you might want to do e.g.
```suggestion
                        return element.label || 'Language: ' + element.lang + ' - Role: ' + element.roles[0];
```
I would _hope_ that a content designer would make the labels unique, and thus wouldn't need the role tacked on to the label.

Reply:
@squarebracket - good suggestion.  updated.